framework/source/class/qx/lang/normalize/Date.js

```diff
@@ -37,13 +37,80 @@ qx.Bootstrap.define("qx.lang.normalize.Date", {
      */
     now : function() {
       return +new Date();
+    },
+
+
+    /**
+     * Parses a string representation of a date and return number of
+     * milliseconds since Epoch or NaN if string is unrecognized.
+     *
+     * <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse">MDN documentation</a>
+     *
+     * Derived from <https://github.com/csnover/js-iso8601>: ©2011 Colin Snover
+     * <http://zetafleet.com>, MIT license
+     *
+     * @param dateString {String} A string representing an RFC2822 or ISO 8601
+     *   date (other formats may be used, but results may be unexpected).
+     * @return {Number|NaN} A number representing the milliseconds elapsed
+     *   since January 1, 1970, 00:00:00 UTC and the date obtained by parsing
+     *   the given string representation of a date.
+     *   If the argument doesn't represent a valid date, NaN is returned.
+     */
+    parse : function (dateString)
+    {
+      // Match input against ISO8601 regular expression
+      var captureGroups = /^(\d{4}|[+\-]\d{6})(?:-(\d{2})(?:-(\d{2}))?)?(?:T(\d{2}):(\d{2})(?::(\d{2})(?:\.(\d{3}))?)?(?:(Z)|([+\-])(\d{2})(?::(\d{2}))?)?)?$/.exec(dateString);
+      if (!captureGroups) {
+        return Date.originalParse(dateString);
+      }
+
+      // Just a date without time?
+      var noTime = [ 4, 5, 6, 7 ].every(function (i) {
+        return captureGroups[i] === undefined;
+      });
+
+      // Avoid invalid timestamps caused by undefined values being passed to Date.UTC
+      [ 1, 4, 5, 6, 7, 10, 11 ].forEach(function (i) {
+        captureGroups[i] = +captureGroups[i] || 0;
+      });
+      captureGroups[2] = (+captureGroups[2] || 1) - 1; // Allow undefined months
+      captureGroups[3] =  +captureGroups[3] || 1;      // Allow undefined days
+
+      // No timezone offset given and *not* just a date (without time)
+      if (captureGroups[8] !== "Z" && captureGroups[9] === undefined && !noTime)
+      { // => Treat as local
+        return new Date( captureGroups[1], captureGroups[2], captureGroups[3],
+                         captureGroups[4], captureGroups[5], captureGroups[6],
+                         captureGroups[7] ).getTime();
+      }
+
+      // Handle timezone offsets
+      var minutesOffset = 0;
+      if (captureGroups[8] !== "Z")
+      {
+        minutesOffset = captureGroups[10] * 60 + captureGroups[11];
+        if (captureGroups[9] === "+") {
+          minutesOffset = - minutesOffset;
+        }
+      }
+
+      // Return the number of milliseconds since Epoch
+      return Date.UTC( captureGroups[1], captureGroups[2], captureGroups[3],
+                       captureGroups[4], captureGroups[5] + minutesOffset,
+                       captureGroups[6], captureGroups[7] );
     }
   },
 
-  defer : function(statics) {
+  defer : function (statics)
```

Comment:
Yeah, that's what I like the best: declaration/definition **with** space, calls **without** space :+1: 
But usually I only apply this to methods that I've changed, not on a whole module. It confuses people when looking at the history (issue-related vs. other changes).